ui/eureka/eureka.js

```diff
@@ -1700,14 +2125,19 @@ class EurekaTracker {
     this.ResetZone();
     this.updateTimesHandle = null;
     this.fateQueue = [];
+    this.CEQueue = [];
   }
 
-  TransByParserLang(obj) {
-    return obj[this.options.ParserLanguage] || obj['en'];
+  TransByParserLang(obj, key) {
+    if (key == null)
```

Comment:
Should probably be strict eq as well
```suggestion
    if (key === null)
```
or falsey check if it's not a guaranteed `null`?
```suggestion
    if (!key)
```